app/controllers/org_admin/plans_controller.rb

```diff
@@ -1,10 +1,10 @@
 module OrgAdmin
   class PlansController < ApplicationController
-    after_action :verify_authorized
-
+    # GET org_admin/plans
     def index
-      authorize Plan
-    
+      # Test auth directly and throw Pundit error sincePundit is unaware of namespacing
+      raise Pundit::NotAuthorizedError unless current_user.present? && current_user.can_org_admin?
```

Comment:
Pundit is unaware of namespacing, however you can still create the class under policies/org_admin/plans_policy.rb. That means, the condition for the unless can be reused in other places too.  You would call the policy as OrgAdmin::PlansPolicy.new(current_user).index? after that unless